plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridTriggerManager.java

```diff
@@ -130,6 +140,29 @@ protected void addObjectModifyActions(
     }
 
     @Override
+    protected void addObjectRenameActions(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull DBCExecutionContext executionContext,
+            @NotNull List<DBEPersistAction> actions,
+            @NotNull ObjectRenameCommand command,
+            @NotNull Map<String, Object> options) {
+        CubridTrigger trigger = (CubridTrigger) command.getObject();

```

Comment:
Closing parentheses must be on their own line
```suggestion
            @NotNull Map<String, Object> options
        ) {
        CubridTrigger trigger = (CubridTrigger) command.getObject();

```

Reply:
@ShadelessFox 
Would you like me to apply this style to all functions, or only to the ones modified in this pull request?

Reply:
I have just applied this style to all functions in Manager class.

Reply:
> Would you like me to apply this style to all functions, or only to the ones modified in this pull request?

Just the modified.